### ISSUE TITLE
Django 4.1.X support coming soon?

### ISSUE THREAD

**Issue by striveforbest** [jazzband/django-redis#653]
**Problem Statement**
According to setup.cfg the latest supported Django version is 4.0. Is supporting Django 4.1.X on the nearest future roadmap?

**Describe the solution you'd like**
Support Django 4.1.X

**Comment by WisdomPill** [comment]
Hello @striveforbest,

where did you see that we don't support django 4.1.x?

Recently I made a couple of PRs to support python 3.11 (#650) and drop old versions of django (#648).

A release is in the pipeline, just need to decide what things to fit into it

**Comment by striveforbest** [comment]
@WisdomPill you're absolutely right, seeing it [here](https://github.com/jazzband/django-redis/blob/master/setup.cfg#L17).
I must have been at some tag, not master. Sorry about it.
